model-engine/model_engine_server/inference/batch_inference/vllm_batch.py

```diff
@@ -414,6 +420,30 @@ async def generate_with_vllm(
         if return_token_log_probs:
             output.tokens = tokens
 
+        tags = [
+            f"model:{model}",
```

Comment:
is this the base model name or finetuned model?

Reply:
base model. there's no name for the finetuned model. i can add a tag for the checkpoint path, but that'll be a very long string and not sure if useful?

Reply:
We do have an `is_finetuned` flag in our metrics. Would it be useful to propagate that here too?

Reply:
sure